django_sorcery/db/query.py

```diff
@@ -53,6 +53,13 @@ class Query(sa.orm.Query):
     A customized sqlalchemy query
     """
 
+    def __init__(self, entities, session=None):
+        ents = entities if isinstance(entities, (list, tuple)) else (entities,)
+        ent_zero = next(iter(ents), None)
+        ent_zero = getattr(ent_zero, "class_", ent_zero)
```

Comment:
maybe use:

```suggestion
self.queryset._only_entity_zero().mapper.class_
```

or if entity might be missing such as doing explicit execute without a model:

```suggestion
self.queryset._entity_zero().mapper.class_
```